package.json

```diff
@@ -26,6 +26,9 @@
   },
   "author": "MX",
   "license": "ISC",
+  "publishConfig": {
```

Comment:
I'm pretty sure `public` is the default. 
https://docs.npmjs.com/cli/v9/using-npm/config#access

Reply:
I was not able to publish the package without this. We had the same issue in Cartographer and use this there as well. I think it may be specific to the package being in an org (mxenabled or mx-cartographer). When I've published packages outside of an org I didn't need this.

Reply:
Ah, good to know.